### ISSUE TITLE
Support for folders in Briefer.

### ISSUE THREAD

**Issue by LucasAlvesDeveloper** [briefercloud/briefer#132]
# Context

So, for some context for this feature request:

I'm a university student in a research project related to Multi-modal saliency models. In this case, we need to analyze data and compare from two sets of data: one where users heard audio and the other where the users didn't hear the audio. Now, the thing is that the two types of data are differentiated by folders, not filenames.

Right now, on the cloud version of briefer, I can only add single files. However, since the data is differentiated by folders, such as "without_sound/_band/user1.csv" and "with_sound/_band/user1.csv", there will be multiple files with the same name, but different data.

Some examples here to demonstrate what i'm talking about:
![image](https://github.com/user-attachments/assets/df71df14-13c0-437a-8d88-b55f7fb95eb7)
![image](https://github.com/user-attachments/assets/ab336021-7081-4a17-88f5-75d639b17eab)

# Feature Request

So, the feature I'm requesting is to add support for folder navigation in briefer, such as adding and removing folders to allow for files with different datasets to be added without the confusion of knowing which one is which, or atleast some sort of grouping for files would be nice.

Also, sorry for the grammar here, English is not my native language and writing about a field I'm not familiar with is kind of hard.

**Comment by lucasfcosta** [comment]
Hey Lucas! Thanks for the issue. I like this idea.

I'd love to hear from others how they use files in Briefer so we can figure out how to spec this.

Anyone with ideas/suggestions is welcome to post here.

**Comment by hiteshwadhwani** [comment]
I agree that adding support for folder navigation or grouping files based on datasets would help reduce confusion when dealing with a large number of files

**Comment by TheFehr** [comment]
It "supports" folders via self-hosting so far as that I could create the folder and put files into it and use that with duckdb.
